docs-source/spring/content/setup/_index.md

```diff
@@ -125,3 +127,34 @@ Oracle Backend as a Service for Spring Cloud setup will install the following co
     <!-- spellchecker-disable -->
     {{< img name="oci-stack-apply-logs" size="large" lazy=false >}}
     <!-- spellchecker-enable -->
+
+## Setup the local workstation
+
+1. Setting Up Cluster Access
+
+    To access a cluster using kubectl installed (see the [Kubernetes access](./cluster-access)) locally. If you haven't already done so,
+
+    - install kubectl (see the [kubectl documentation](https://kubernetes.io/docs/tasks/tools/install-kubectl/)).
+    - generate an API signing key pair
+        If you already have an API signing key pair, go straight to the next step. If not:
+
+        1. Use OpenSSL commands to generate the key pair in the required PEM format. If you're using Windows, you'll need to install Git Bash for Windows and run the commands with that tool. See [How to Generate an API Signing Key](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/apisigningkey.htm#two).
+        2. Copy the contents of the public key to the clipboard (you'll need to paste the value into the Console later).
+
+    - add the public key value of the API signing key pair to the User Settings for your username
+       1. In the top-right corner of the Oracle Cloud Infrastructure Console, open the **Profile** menu (User menu icon) and then click **User Settings** to view the details.
+       2. Click **Add Public Key**.
+       3. Paste the public key's value into the window and click **Add**.
+
+          The key is uploaded and its fingerprint is displayed (for example, d1:b2:32:53:d3:5f:cf:68:2d:6f:8b:5f:77:8f:07:13).
```

Comment:
i assume this is a made-up fingerprint?

Reply:
yes.. it was copied from official docs.
https://docs.oracle.com/en-us/iaas/Content/ContEng/Tasks/contengdownloadkubeconfigfile.htm#